src/components/utils/ErrorHandler.js

```diff
@@ -0,0 +1,224 @@
+import GridLabelValue from "./GridLabelValue";
+import { useUserProfile } from "../../contexts/userProfile";
+import { build, getMessage, withI18N } from "@cyverse-de/ui-lib";
+import messages from "./messages";
+import ids from "./ids";
+import {
+    Button,
+    Card,
+    CardActions,
+    CardContent,
+    CardHeader,
+    Divider,
+    Grid,
+    Link,
+    makeStyles,
+    Typography,
+} from "@material-ui/core";
+
+import LiveHelpIcon from "@material-ui/icons/LiveHelp";
+import ErrorIcon from "@material-ui/icons/Error";
+import constants from "../../constants";
+
+const useStyles = makeStyles((theme) => ({
+    cardContent: {
+        padding: theme.spacing(1),
+        [theme.breakpoints.down("xs")]: {
+            padding: theme.spacing(0),
+        },
+    },
+    container: {
+        margin: theme.spacing(1),
+        [theme.breakpoints.down("xs")]: {
+            margin: theme.spacing(0),
+        },
+    },
+    signIn: {
+        padding: theme.spacing(1),
+    },
+}));
+
+function ClientInfo(props) {
+    const { baseId } = props;
+    const browser = Bowser.getParser(window.navigator.userAgent);
+    const [userProfile] = useUserProfile();
+    return (
+        <>
+            <GridLabelValue label={getMessage("user")}>
+                <span id={build(baseId, ids.USER)}>{userProfile?.id}</span>
+            </GridLabelValue>
+            <GridLabelValue label={getMessage("browser")}>
+                <span id={build(baseId, ids.BROWSER)}>
+                    {browser.getBrowser().name} - {browser.getBrowser().version}
+                </span>
+            </GridLabelValue>
+            <GridLabelValue label={getMessage("os")}>
+                <span id={build(baseId, ids.OS)}>
+                    {browser.getOS().name} - {browser.getOS().versionName} -
+                    {browser.getOS().version}
+                </span>
+            </GridLabelValue>
+            <GridLabelValue label={getMessage("host")}>
+                <span id={build(baseId, ids.host)}>{window.location.origin}</span>
+            </GridLabelValue>
+        </>
+    );
+}
+
+function ContactSupport(props) {
+    const { baseId } = props;
+    return (
+        <Button
+            id={build(baseId, ids.CONTACT_SUPPORT_BUTTON)}
+            color="primary"
+            startIcon={<LiveHelpIcon />}
+            onClick={() => window.Intercom("show")}
+        >
+            {getMessage("contactSupport")}
+        </Button>
+    );
+}
+
+function ErrorHandler(props) {
+    const { errorObject, baseId } = props;
+    const router = useRouter();
+    const classes = useStyles();
+    const errBaseId = build(baseId, ids.ERROR_HANDLER);
+    if (!errorObject || !errorObject.response) {
+        return (
+            <Card id={errBaseId}>
+                <CardHeader
+                    title={
+                        <Typography color="error" variant="h6">
+                            <ErrorIcon /> {getMessage("oops")}
```

Comment:
At least in the screenshots, the `i` icons seem to be raised higher than the text?  Is there a way to fix it so that it's level with the text?

Reply:
I was trying to but I couldn't..I will try again.